lib/goth/token.ex

```diff
@@ -32,8 +32,8 @@ defmodule Goth.Token do
 
     * `:source` - See "Source" section below.
 
-    * `:http_client` - HTTP client configuration, defaults to using `Goth.HTTPClient.Hackney`.
-      See `Goth.HTTPClient` for more information.
+    * `:http_client` - HTTP client configuration, defaults to using `{&Goth.__hackney__/1, []}`.
+      See `Goth` for more information.
```

Comment:
notice Goth.start_links/1 tells users to check out :source option in Goth.Token.fetch. Thus we shouldn't say in Goth.Token.fetch to see Goth.start_link/1 to check :http_client option. We need to pick a module which should give the most information. Let's stick to giving more docs on Goth.Token.

Reply:
Sure, I'll add more detailed docs + examples. WDYT?

Reply:
sounds good